### ISSUE TITLE
Fix certificate bot

### ISSUE THREAD

**Issue by mxplusb** [18F/cg-product#1102]
In order to be notified of expiring certificates, the certificate bot needs to work.

## Acceptance Criteria
* The certificate bot doesn't use webhooks.
* The certificate bot can read certificates from Credhub.
* The certificate bot can post to slack.

--- 

## Security considerations

This has to be a deployed app. I'd prefer not to have this be a bosh release, but we'll see what happens.

**Comment by jontours** [comment]
Also @mxplusb important to note, the bot will have to check both instance groups like it did before _and_ check credhub.  Since secrets are still not all moved over yet.  This might be possible with `credhub interpolate`.

**Comment by karareinsel** [comment]
@mxplusb This is a low priority. I'd like to move this out of the current sprint so that higher priority work that the team has planned for can be addressed.

**Comment by karareinsel** [comment]
Aim to include in sprint 19.11.29.

**Comment by mxplusb** [comment]
This needs to get fixed, but I think doomsday is a good replacement, so we can just use webhooks for that integration.

**Comment by karareinsel** [comment]
Revisit this once the webhooks (#1155 )work is complete.

**Comment by mxplusb** [comment]
I think we just need to add the current webhook to Doomsday and then this can get closed out.
